src/Microsoft.ApplicationInsights/Channel/Transmission.cs

```diff
@@ -200,7 +200,7 @@ public virtual async Task<HttpWebResponseWrapper> SendAsync()
                             }
                         }
                     }
-                    catch (TaskCanceledException)
+                    catch (OperationCanceledException)
```

Comment:
hi - can you explain this change? Tests were passing with TaskCancelledException, so not sure why we need to use its base class here.

Reply:
I tried a sample .netcorreapp2.1 app and it is throwing `TaskCanceledException`. But yes, its right to catch the base exception.